com.unity.mobile.notifications/Editor/NotificationSettings.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace Unity.Notifications
 {
+    /// <summary>
+    /// Class used to access notification settings for a specific platform.
+    /// </summary>
     public class NotificationSettings
```

Comment:
Checked today that these notification settings classes can be internal.

Again, we should do this in a minor release.

Reply:
WIll take note